### DISCUSSION TITLE
突然无法翻译了，更新到最新版本后出现了"unknown error"的报错

### DISCUSSION THREAD

**Discussion by CFionaBF** [nextai-translator/nextai-translator#662] [Q&A]
如题，请问，有什么解决方法吗？谢谢！
<img width="602" alt="error" src="https://user-images.githubusercontent.com/72554686/235647763-2ef4346e-3f30-46f3-b461-cb8adeffc2e2.png">

**Comment by yetone** [comment]
是 OpenAI 偷偷下线了几个模型导致的，去设置页面选择一个可用的 model 即可，新版本会有详细的错误提示

<img width="779" alt="image" src="https://user-images.githubusercontent.com/1206493/235789938-ec65a5dd-0809-42f2-b7b3-3a8064484269.png">

**Comment by FYboss** [comment]
换了好多个模型都还是Unknown error，github的api
![image](https://github.com/user-attachments/assets/7973380d-ee57-4820-863f-d16e25ae1869)
